tests/unit/Cassandra/TimeUuidTest.php

```diff
@@ -60,36 +62,37 @@ public function notEqualTypes()
      */
     public function testInitFromStringType1()
     {
-        new TimeUuid('5f344f20-52a3-11e7-915b-5f4f349b532d');
+        $uuid = new TimeUuid('5f344f20-52a3-11e7-915b-5f4f349b532d');
+        $this->assertIsObject($uuid);
     }
 
     /**
      * TimeUuid cannot be created from UUID type 4
-     * @expectedException         Cassandra\Exception\InvalidArgumentException
-     * @expectedExceptionMessage  UUID must be of type 1, type 4 given
      */
     public function testInitFromStringType4()
     {
+        $this->expectException(\Cassandra\Exception\InvalidArgumentException::class);
+        $this->expectExceptionMessage("UUID must be of type 1, type 4 given");
         new TimeUuid('65f9e722-036a-4029-b03b-a9046b23b4c9');
     }
 
     /**
      * TimeUuid cannot be created from invalid string
-     * @expectedException         Cassandra\Exception\InvalidArgumentException
-     * @expectedExceptionMessage  Invalid UUID
      */
     public function testInitFromInvalidString()
     {
+        $this->expectException(\Cassandra\Exception\InvalidArgumentException::class);
+        $this->expectExceptionMessage("Invalid UUID");
         new TimeUuid('invalid');
     }
 
     /**
      * TimeUuid requires string or integer in constructor
-     * @expectedException         Cassandra\Exception\InvalidArgumentException
-     * @expectedExceptionMessage  Invalid argument
      */
     public function testInitInvalidArgument()
     {
+        $this->expectException(\Cassandra\Exception\InvalidArgumentException::class);
+        $this->expectExceptionMessage("Invalid argument");
         new TimeUuid(new \Datetime());
     }
```

Comment:
Did you try to run our internal timeuuid tests over this new implementation?

Reply:
Yes of course : https://github.com/rgsystemes/rgsupv-core/pull/4423#discussion_r816773059
I've been pushing a [php8.1 core docker image](https://github.com/rgsystemes/ansible-roles/pkgs/container/ansible-roles%2Fcore/16920313?tag=php8.1) containing the [ext-cassandra:1.4.0](https://github.com/qkdreyer/rgsupv-core/blob/e2e773233883f6a97d34c981278218555a87510f/composer.json#L9)